trio/__init__.py

```diff
@@ -13,86 +13,101 @@
+    sleep as sleep,
+    fail_at as fail_at,
+    fail_after as fail_after,
+    TooSlowError as TooSlowError,
 )
 
 from ._sync import (
-    Event,
-    CapacityLimiter,
-    Semaphore,
-    Lock,
-    StrictFIFOLock,
-    Condition,
+    Event as Event,
+    CapacityLimiter as CapacityLimiter,
+    Semaphore as Semaphore,
+    Lock as Lock,
+    StrictFIFOLock as StrictFIFOLock,
+    Condition as Condition,
 )
 
-from ._highlevel_generic import aclose_forcefully, StapledStream
+from ._highlevel_generic import (
+    aclose_forcefully as aclose_forcefully,
+    StapledStream as StapledStream,
+)
 
 from ._channel import (
-    open_memory_channel,
-    MemorySendChannel,
-    MemoryReceiveChannel,
+    open_memory_channel as open_memory_channel,
+    MemorySendChannel as MemorySendChannel,
+    MemoryReceiveChannel as MemoryReceiveChannel,
 )
 
-from ._signals import open_signal_receiver
+from ._signals import open_signal_receiver as open_signal_receiver
 
-from ._highlevel_socket import SocketStream, SocketListener
+from ._highlevel_socket import (
+    SocketStream as SocketStream,
+    SocketListener as SocketListener,
+)
 
-from ._file_io import open_file, wrap_file
+from ._file_io import open_file as open_file, wrap_file as wrap_file
 
-from ._path import Path
+from ._path import Path as Path
 
-from ._subprocess import Process, run_process
+from ._subprocess import Process as Process, run_process as run_process
 
-from ._ssl import SSLStream, SSLListener, NeedHandshakeError
+from ._ssl import (
+    SSLStream as SSLStream,
+    SSLListener as SSLListener,
+    NeedHandshakeError as NeedHandshakeError,
+)
 
-from ._dtls import DTLSEndpoint, DTLSChannel
+from ._dtls import DTLSEndpoint as DTLSEndpoint, DTLSChannel as DTLSChannel
 
-from ._highlevel_serve_listeners import serve_listeners
+from ._highlevel_serve_listeners import serve_listeners as serve_listeners
 
-from ._highlevel_open_tcp_stream import open_tcp_stream
+from ._highlevel_open_tcp_stream import open_tcp_stream as open_tcp_stream
 
-from ._highlevel_open_tcp_listeners import open_tcp_listeners, serve_tcp
+from ._highlevel_open_tcp_listeners import (
+    open_tcp_listeners as open_tcp_listeners,
+    serve_tcp as serve_tcp,
+)
 
-from ._highlevel_open_unix_stream import open_unix_socket
+from ._highlevel_open_unix_stream import open_unix_socket as open_unix_socket
 
 from ._highlevel_ssl_helpers import (
-    open_ssl_over_tcp_stream,
-    open_ssl_over_tcp_listeners,
-    serve_ssl_over_tcp,
+    open_ssl_over_tcp_stream as open_ssl_over_tcp_stream,
+    open_ssl_over_tcp_listeners as open_ssl_over_tcp_listeners,
+    serve_ssl_over_tcp as serve_ssl_over_tcp,
 )
 
 from ._core._multierror import MultiError as _MultiError
 from ._core._multierror import NonBaseMultiError as _NonBaseMultiError
 
-from ._deprecate import TrioDeprecationWarning
+from ._deprecate import TrioDeprecationWarning as TrioDeprecationWarning
 
 # Submodules imported by default
```

Comment:
Do submodules not need the `as ...` dance?

Reply:
They don't have a leading underscore so they're considered public by default https://microsoft.github.io/pyright/#/typed-libraries?id=library-interface